webapp/channels/src/components/advanced_text_editor/advanced_text_editor.scss

```diff
@@ -181,20 +180,18 @@
             }
         }
 
-        &--has-error {
```

Comment:
It looks like we're not using this class elsewhere, so would you mind removing that from [this line](https://github.com/mattermost/mattermost/blob/master/webapp/channels/src/components/advanced_text_editor/advanced_text_editor.tsx#L757) by changing it to `className='AdvancedTextEditor__footer'`?